### DISCUSSION TITLE
settings of parallel optimization of gurobi

### DISCUSSION THREAD

**Discussion by BYS543** [yalmip/YALMIP#924] [General discussion]
Hi, Johan
Recently, I deal with a  large model. So I want to set the parallel calculation parameters to parallel it. I find `distrbutedMIPJobs` parameter(the default value is 0). So can I set which machines I can use for parallel calculation in MATLAB?
Thanks a lot!

**Comment by johanlofberg** [comment]
No idea. You will have to try
